bulbs/special_coverage/models.py

```diff
@@ -107,15 +104,12 @@ def _save_percolator(self):
             return
 
         # We'll need this data, to decide which special coverage section to use
-        q["sponsored"] = bool(self.campaign)
+        q["sponsored"] = bool(self.tunic_campaign_id)
         # Elasticsearch v1.4 percolator "field_value_factor" does not
         # support missing fields, so always need to include
-        if self.campaign:
-            q["start_date"] = self.campaign.start_date
-            q["end_date"] = self.campaign.end_date
-        else:
-            q["start_date"] = self.start_date
-            q["end_date"] = self.end_date
+
+        q["start_date"] = self.start_date
+        q["end_date"] = self.end_date
```

Comment:
I believe @spra85 said for the time being we are just going to rely on them enforcing the start/end dates on the special coverage object iself